python/cugraph/tests/test_jaccard.py

```diff
@@ -222,3 +223,32 @@ def test_jaccard_nx(graph_file):
     # FIXME:  Nx does a full all-pair Jaccard.
     # cuGraph does a limited 1-hop Jaccard
     # assert nx_j == cg_j
+
+
+@pytest.mark.parametrize("graph_file", utils.DATASETS_UNDIRECTED)
+def test_jaccard_multi_column(graph_file):
+    gc.collect()
+
+    M = utils.read_csv_for_nx(graph_file)
+
+    cu_M = cudf.DataFrame()
+    cu_M["src_0"] = cudf.Series(M["0"])
+    cu_M["dst_0"] = cudf.Series(M["1"])
+    cu_M["src_1"] = cu_M["src_0"] + 1000
+    cu_M["dst_1"] = cu_M["dst_0"] + 1000
```

Comment:
Just wondering why two different patterns to create `cu_M` (test_force_atlas2  and test_jaccacrd). I would probably stick with one for consistency.  During refactoring the generation of a graph from cu_M will definitely be in a fixture file to avoid rebuilding the same graph multiple times when running multiple test algo at once (just a side comment)

Reply:
updated for consistency